pkg/operator/ceph/client/controller.go

```diff
@@ -257,6 +257,8 @@ func (r *ReconcileCephClient) createOrUpdateClient(cephClient *cephv1.CephClient
 		},
 		StringData: map[string]string{
 			cephClient.Name: key,
+			"userID":        key,
+			"userKey":       key,
```

Comment:
`userID` and `userKey` will work for RBD, not cephfs. For cephfs, the secrets data contains `adminID` and `adminKey` https://github.com/rook/rook/blob/e392dbf9e8e27d1cb6bca41b954fed1189189bec/pkg/operator/ceph/csi/secrets.go#L115-L137

So the change will be as follows.

```suggestion
// CSI requires userID and userKey for RBD 
"userID":        cephClient.Name,
"userKey":       key,

// CSI requires adminID and adminKey for CephFS 

"adminID":        cephClient.Name,
"adminKey":       key,
```

Reply:
Yes, I checked yesterday there can be 2 users, RBD & CephFS, will update. Thanks!